tools/tiled_ensemble/predictions/basic_joiner.py

```diff
@@ -0,0 +1,176 @@
+"""Concrete implementation of basic prediction joiner."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from typing import Dict, List
+
+import torch
+from tools.tiled_ensemble.predictions.prediction_joiner import EnsemblePredictionJoiner
+from torch import Tensor
+
+
+class BasicPredictionJoiner(EnsemblePredictionJoiner):
```

Comment:
It would be good to explain to the users why we need the basic implementation, or why is it named Basic

Reply:
The naming might even need to be adusted. The idea of it being basic is that it just joins the data, it doesn't perform any modification operations on the data except the averaging. Should the name be changed or do I write this clarification to docstring?